### PR TITLE
Rename to ForemanGoogle

### LINKED ISSUE #2: Name of the plugin
@stejskalleos didn't we speak about naming the plugin `foreman_google`?

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: lib/tasks/foreman_gce_tasks.rake [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'rake/testtask'
```
Comment (ezr-ondrej) [inline lib/tasks/foreman_gce_tasks.rake:1]: We should change the name of the file, right?

**Review by ezr-ondrej — COMMENTED** [review]
Some files have still `gce` in the name

**Comment by stejskalleos** [comment]
> Some files have still `gce` in the name

Fixed

**Review by ezr-ondrej — APPROVED** [review]
LGTM :+1:

**Pull request merged**